.gitignore

```diff
@@ -2,3 +2,5 @@
 /build*/
 /site/
 bazel-*
+# Bzlmod lockfile
+MODULE.bazel.lock
```

Comment:
I would guess that the file can only be present in the project root only.
```suggestion
/MODULE.bazel.lock
```

Reply:
done